src/Exceptions/HttpNotAcceptable.php

```diff
@@ -8,4 +8,3 @@ class HttpNotAcceptable extends HttpException {
     public const DESCRIPTION = "Indicates that the target resource does not have a current representation that would be acceptable to the user agent, according to the proactive negotiation header fields received in the request, and the server is unwilling to supply a default representation.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces